auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/LdapService.kt

```diff
@@ -44,9 +46,9 @@ class LdapService(private val config: Configuration) {
                 arrayOf("cn", "memberOf", "mail", "unixHomeDirectory", "givenName", "sn", "userAccountControl")
             )
 
-        val cn = attributes.get("cn").get() as String? ?: throw InvalidLdapUserException("No value for attribute cn")
+        val cn = attributes.get("cn")?.get() as String? ?: throw InvalidLdapUserException("No value for attribute cn")
         val email =
-            attributes.get("mail").get() as String? ?: throw InvalidLdapUserException("No value for attribute mail")
+            attributes.get("mail")?.get() as String? ?: throw InvalidLdapUserException("No value for attribute mail")
```

Comment:
As mentioned on Slack - does the user really need a mail attribute? E.g. service users.